apps/expo-go/src/components/UserReviewSection.tsx

```diff
@@ -78,8 +78,8 @@ export default function UserReviewSection({ snacks, apps }: Props) {
 const styles = StyleSheet.create({
   dismissButton: {
     position: 'absolute',
-    top: 8,
-    right: 8,
+    top: spacing[4],
+    right: spacing[4],
```

Comment:
I styled the prompt so it would look similar to the user review prompt (as there are cases where both could be displayed) and noticed that the close button was not aligned with the header so updated both to match.